### PR TITLE
[BEAM-3753] Fix Flink Runner integration tests

### PR DESCRIPTION
**Pull request by grzegorz8**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand:
   - [ ] What the pull request does
   - [ ] Why it does it
   - [ ] How it does it
   - [ ] Why this approach
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

File: runners/flink/build.gradle [hunk 1]
```diff
@@ -72,6 +72,7 @@ dependencies {
   shadowTest library.java.junit
   shadowTest library.java.mockito_core
   shadowTest library.java.google_api_services_bigquery
+  shadowTest project(path: ":sdks:java:io:google-cloud-platform", configuration: "shadow")
```
Comment (aljoscha) [inline runners/flink/build.gradle]: Why is it `"shadow"` and not `"shadowTest"`? I must admit I don't have much experience with Gradle.

**Comment by aljoscha** [comment]
Run Flink ValidatesRunner

File: runners/flink/build.gradle (hunk 1 above)
Comment (lukecwik) [reply runners/flink/build.gradle]: `shadow` is correct.

I wrote this up to help people when choosing configurations.
https://github.com/apache/beam/blob/5b99fe2ea44e1a0d8a6f802ca757d1a1cfab91cc/build_rules.gradle#L150

**Comment by aljoscha** [comment]
@grzegorz8 If you want, you could open an issue for setting the uid on Flink operators. (see https://ci.apache.org/projects/flink/flink-docs-master/ops/production_ready.html#set-uuids-for-operators)

**Pull request merged**